clients/src/test/java/org/apache/kafka/common/metrics/MetricsTest.java

```diff
@@ -424,6 +424,14 @@ public void testPercentiles() {
         assertEquals(0.0, p25.value(), 1.0);
         assertEquals(0.0, p50.value(), 1.0);
         assertEquals(0.0, p75.value(), 1.0);
+
+        // record two more windows worth of sequential values
```

Comment:
This test sets eventWindow to 50 ([line 406 above](https://github.com/iv-m/kafka/blob/07c176a23ea11b6ba7a103d8aaf4867591ba9293/clients/src/test/java/org/apache/kafka/common/metrics/MetricsTest.java#L406)), so we don't need to adjust the time -- it's enough to record 50 events to roll one sample, and 100 events to roll all the two of them. That's how the test works in trunk now btw -- I'm just adding one more "full roll" that reproduces the issue/confirms the fix.